### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by OdongAlican**
# This is milestone2 of the React Bookstore project.
## In this milestone, we did the following:-

- Created components and containers directories in the src
- Created an actions directory with the index.js
- Created a reducers directory with two files: books.js and index.js
- Added BooksList and BooksForm in the App component
- Update BooksList, Book, and BooksForm components

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,46 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
 import App from './App';
+import rootReducer from './reducers/index';
+
+const initialState = {
+  book: [
```
Comment (suretrust) [inline src/index.js]: Perhaps, it would be better if this is plural?
```suggestion
  books: [
```

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi team, 👋 

### Great job thus far! 🙌 Please see my comments. 👇 

- [x] The `App` component should be placed in `src/components` directory.
- [x] The `Book` component should be placed in `src/components` directory.
- [x] The `BookList` component should be placed in `src/containers` directory.
- [ ] The `BookForm` component should be placed in `src/containers` directory.

**Note:** Any component that connects to the store should be placed in the `containers` directory.

**KIndly submit another CRR once these issues have been solved.** _Happy coding!_ 🎉

**Review by Oghenebrume50 — APPROVED** [review]
### APPROVED

Hi @OdongAlican good job you have done so far, kudos for making changes as earlier requested, this looks good to me :+1: 
Please merge and move on, **This is approved** :slightly_smiling_face: 

**RAPHAEL NORIODE** :mask: STAY_SAFE

**Comment by jaspreet-singh-sahota** [comment]
hi, @suretrust @Oghenebrume50,

Thank you for the review and for approving the code.

best regards,
Jaspreet Singh

**Pull request merged**